packages/yargs/mixins/log/mixin.core.js

```diff
@@ -44,13 +44,8 @@ module.exports = class CLIMixin extends Mixin {
   }
   configureServer(app, middlewares, mode) {
     if (mode !== 'static') {
-      const morgan = require('morgan');
-      const split = require('split');
-      app.use(
-        morgan('tiny', {
-          stream: split().on('data', (line) => this.logger.request(line)),
-        })
-      );
+      const { loggerMiddleware } = require('../../lib/request');
+      app.use(loggerMiddleware(this.logger));
```

Comment:
So, you said it's only for development, but won't this include it also for `mode === 'serve'`?

Reply:
Yup. Nothing prevents you from calling `un serve -pv`, but personally, I would prefer another logging solution. In production, I would rather not have app servers log anything request related themselves, t.b.h.

Reply:
So we should guard this with `process.env.NODE_ENV !== 'production'` right?